packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -56,19 +61,60 @@ export const TokensList: React.FC<TokensListProps> = (props) => {
         }}
       />
       <Content>
-        <Box paddingBottom="space80">
-          <Label htmlFor="test" id="test-label">
-            Filter tokens
-          </Label>
-          <Input
-            type="text"
-            id="test"
-            aria-labelledby="test-label"
-            onChange={handleInput}
-            insertBefore={<FilterIcon decorative={false} title="Description of icon" />}
-          />
+        <Box marginBottom="space80">
+          <Grid gutter="space40">
+            <Column span={6}>
+              <Label htmlFor="test" id="test-label">
+                Filter tokens
+              </Label>
+              <Input
+                type="text"
+                id="test"
+                aria-labelledby="test-label"
+                onChange={handleInput}
+                insertBefore={<FilterIcon decorative={false} title="Description of icon" />}
+                placeholder="Filter by token name or value"
+              />
+            </Column>
+            <Column span={3}>
+              <Label htmlFor="theme-control" id="theme-control-label">
+                Theme
+              </Label>
+              <Select
+                id="theme-control"
+                defaultValue={theme === 'dark' ? 'dark' : 'default'}
+                onChange={(evt) => {
+                  if (evt.target.value === 'dark') {
+                    setSelectedTheme('dark');
+                    setTokens(DarkModeTokens.tokens);
```

Comment:
question: Should we be thinking about these as separate, independent pieces of state, or should we start thinking about them being tightly coupled?

Reply:
good point, I think I'm actually removing the `selectedTheme` state though, because it was just being used to determine the background color of the example in the `TokenCard` so not really necessary anymore